package.json

```diff
@@ -21,10 +20,10 @@
     "url": "https://github.com/srujithm/react-advanced-gauge-chart"
   },
   "dependencies": {
-    "d3": "^5.12.0"
+    "d3": "^7.7.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
Could you please introduce a new script for legacy ssl instead of overwriting the existing one? When I try on mac, this is throwing a failure which might be the case for other developers as well.